docs/quick_start.rst

```diff
@@ -230,27 +236,124 @@ Connect the new application to the OMF Connector Relay by selecting the new Fled
 
 Finally, select the new Fledge application. Click "More" at the bottom of the Configuration panel. Make note of the Producer Token and Relay Ingress URL.
 
-Now go to the Fledge user interface, create a new North instance and select the “pi_server” plugin on the first screen.
+Now go to the Fledge user interface, create a new North instance and select the “OMF” plugin on the first screen.
+The second screen will request the following information:
+
++---------------------+
+| |omf_plugin_config| |
```

Comment:
This image has with endpoint `OSI Cloud Services` selected. It should be with `Connector Relay`